secret_handlers.go

```diff
@@ -120,3 +120,38 @@ func SecretGet(c echo.Context) error {
 
 	return c.JSON(http.StatusOK, secret.ToSecretResponse())
 }
+
+func SecretEdit(c echo.Context) error {
+	secretDao, err := getSecretDao(c)
+	if err != nil {
+		return err
+	}
+
+	updateRequest := &m.SecretEditRequest{}
+	err = c.Bind(updateRequest)
+	if err != nil {
+		return util.NewErrBadRequest(err)
+	}
```

Comment:
here you can return only `err` because our Bind method returns Bad Request err objects